src/main/java/software/amazon/awssdk/crt/Process.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.crt;
+
+import static software.amazon.awssdk.crt.CRT.awsErrorString;
+import static software.amazon.awssdk.crt.CRT.awsLastError;
+
+/**
+ * Encapsulates Process information and manipulation of process level operations.
+ */
+public class Process {
```

Comment:
final?

Reply:
not quite yet, but likely. Also I didn't see this til after I clicked the merge button.